tox.ini

```diff
@@ -4,6 +4,7 @@ envlist = {,ci-}py{36,37,38,39},fuzz
 [testenv]
 setenv = PYTHONPATH = {toxinidir}/src
 skip_install = True
+recreate = True
```

Comment:
I think this could use a comment explaining why we do this and linking back to the issue.

Reply:
that's a good point, thanks!